src/redux/missions/missions.js

```diff
@@ -1 +1,22 @@
-/* some comment */
+const GET_MISSIONS = 'missionStore/missions/GET_MISSIONS';
+
+const initialState = [];
+
+export const getMissions = (payload) => ({
+  type: GET_MISSIONS, payload,
+});
+
```

Comment:
[Optional]
Below this action, you can create a thunk function(action) that will actually call your async fetchMission() function and pass results to this getMissions action. I think this way it will be easier to use it in the useEffect() hook. 
It could look something like this:

```
const getMissionsThunk = () => async (dispatch) => {
  const missions= await fetchMissions();
  if (missions) dispatch(getMissions(missions));
};
```